src/core/lib/security/credentials/tls/grpc_tls_credentials_options.h

```diff
@@ -27,6 +27,21 @@
 #include "src/core/lib/gprpp/ref_counted.h"
 #include "src/core/lib/security/security_connector/ssl_utils.h"
 
+struct grpc_tls_err_details
+    : public grpc_core::RefCounted<grpc_tls_err_details> {
+ public:
+  grpc_tls_err_details() : error_details_("") {}
+  void set_err_details(const char* err_details) {
+    error_details_ = err_details;
+  }
+  const std::string& err_details() { return error_details_; }
```

Comment:
nit. please change the field name `error_details_` to `err_details` to make it consistent with the getter and setter function names.

Reply:
I changed all the name to `error_details`. Now the name should be consistent across files.